### PR TITLE
Add descriptions of environment variables

### PR DESCRIPTION
**Pull request by majecty**

### CODE REVIEW

File: monitor/README.md [hunk 1]
```diff
@@ -1,10 +1,22 @@
...
-* (error level) Checks if CodeChain is alive every hour (whether blocks were created for the past hour or not).
-* (error level) Sends a report if the requested block could not be obtained.
-* (warn level) Sends an alert when the consensus process is not going smoothly and is higher than View's preset View alert level (currently set to 5).
-* (warn level) Sends a report if the observing node sees nodes that did not participate in votes.
-* (warn level) Sends a report if the observing node sees that a node has not been signed for N consecutive blocks (currently 5).
-* (info level) Sends a report when nodes that have slept in succession (votes not being delivered) return to normal voting.
-* (info level) Sends a report if all nodes are working fine and return to a state of agreement.
-* (info level) A notification is sent everyday, reporting that CodeChain is in working order.
\ No newline at end of file
+1 (error level) Checks if CodeChain is alive every hour (whether blocks were created for the past hour or not).
```
[the remark this answers is not in the document]
Comment (majecty) [reply monitor/README.md]: Thanks. I'll fix it.

[the remark this answers is not in the document]
Comment (majecty) [reply monitor/README.md]: I fixed it.

File: auto-self-nominate/README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+### Auto self nominate
+
+Automatically send nominate transaction.
```
Comment (ScarletBlue) [inline auto-self-nominate/README.md]: nominate transactions?

File: auto-self-nominate/README.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+### Environment variables
+
+| Key                             | Description                                                                              | Example values                              | Default value |
+| ------------------------------- | ---------------------------------------------------------------------------------------- | ------------------------------------------- | ------------- |
+| NETWORK_ID                      | Network ID of connected CodeChain                                                        | cc, bc, tc                                  | null          |
+| RPC_URL                         | CodeChain HTTP JSON RPC URL                                                              | http://localhost:8080                       | null          |
+| NEED_NOMINATION_UNDER_TERM_LEFT | Send nomination when nomination will be ended after NEED_NOMINATION_UNDER_TERM_LEFT term | 2                                           | 2             |
```
Comment (ScarletBlue) [inline auto-self-nominate/README.md]: Send nomination if the nomination will end after the NEED_NOMINATION_UNDER_TERM_LEFT term

File: auto-self-nominate/README.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+| ACCOUNT_ADDRESS                 | Platform address of sender                                                               | bccqy7aeyc0qryehxgtx9y4vthhg7u95rad2ymlya7j | null          |
+| PASSPHRASE                      | A passphrase that is used to encrypt the ACCOUNT_ADDRESS                                 | xxxx                                        | null          |
+| METADATA                        | A metadata will be writen in a nomination transaction                                    | Some advertisement text                     | null          |
```
Comment (ScarletBlue) [inline auto-self-nominate/README.md]: I think we don't need 'A' in front of metadata.

a nomination transaction -> nomination transactions

File: auto-self-nominate/README.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+| TARGET_DEPOSIT                  | The goal deposited CCC value of the ACCOUNT_ADDRESS                                      | 10000000                                    | null          |
```
Comment (ScarletBlue) [inline auto-self-nominate/README.md]: The goal deposit CCC value of the ACCOUNT_ADDRESS

File: auto-self-nominate/README.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+| INTERVAL_SECONDS                | This script runs every INTERVAL_SECONDS                                                  | 600                                         | 0             |
+| SLACK_WEBHOOK_URL               | A webhook URL for alarm                                                                  | http://abcd.example.com/x/y                 | null          |
```
Comment (ScarletBlue) [inline auto-self-nominate/README.md]: A webhook URL for alarms

File: auto-self-nominate/README.md [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+| SENDGRID_TO                     | An email address that alarm will be sent to                                              | codechain@example.com                       | null          |
```
Comment (ScarletBlue) [inline auto-self-nominate/README.md]: An email address that alarms will be sent to

File: auto-self-nominate/README.md [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
+| SENDGRID_API_KEY                | An API Key for alarm                                                                     | xxx                                         | null          |
```
Comment (ScarletBlue) [inline auto-self-nominate/README.md]: An API Key for alarms

File: fee-monitor/README.md [hunk 9]
```diff
@@ -1,3 +1,15 @@
...
\ No newline at end of file
+* (warn level) Each block is checked to see if the CCC difference between the previous and current block matches the spec logic calculated by the fee monitor itself.
+
+### Environment variables
+
+| Key               | Description                                            | Example values              | Default value                                                               |
+| ----------------- | ------------------------------------------------------ | --------------------------- | --------------------------------------------------------------------------- |
+| SERVER            | The name of the network                                | beagle OR corgi OR mainnet  | corgi                                                                       |
+| RPC_URL           | CodeChain HTTP JSON RPC URL                            | http://localhost:8080       | https://corgi-rpc.codechain.io in corgi https://rpc.codechain.io in mainnet |
+| SLACK_WEBHOOK_URL | A webhook URL for alarm                                | http://abcd.example.com/x/y | null                                                                        |
```
Comment (ScarletBlue) [inline fee-monitor/README.md]: alarms

File: fee-monitor/README.md [hunk 10]
```diff
@@ -1,3 +1,15 @@
...
+| SENDGRID_TO       | An email address that alarm will be sent to            | codechain@example.com       | null                                                                        |
```
Comment (ScarletBlue) [inline fee-monitor/README.md]: alarms

File: fee-monitor/README.md [hunk 11]
```diff
@@ -1,3 +1,15 @@
...
+| SENDGRID_API_KEY  | An API Key for alarm                                   | xxx                         | null                                                                        |
```
Comment (ScarletBlue) [inline fee-monitor/README.md]: alarms

File: fee-monitor/README.md [hunk 12]
```diff
@@ -1,3 +1,15 @@
...
+| BLOCK_NUMBER      | Start monitoring from $BLOCK_NUMBER                    | 3                           | null                                                                        |
+| LOOK_BEHIND       | Start monitoring from best block number - $LOOK_BEHIND | 3                           | null                                                                        |
```
Comment (ScarletBlue) [inline fee-monitor/README.md]: Start monitoring from the best block number

File: indexer-monitor/README.md [hunk 13]
```diff
@@ -6,3 +6,12 @@ Checks whether the indexer is working properly.
 * (info level) Sends a report when the indexer's synchronization status returns to normal.
 * (info level) Sends a report when the CodeChain node that the indexer is watching is normal.
 * (info level) Sends a report that notifies whether the indexer is running once a day.
+
+| Key               | Description                                 | Example values              | Default value |
+| ----------------- | ------------------------------------------- | --------------------------- | ------------- |
+| NODE_ENV          | Use "production" to reduce mistakes         | production OR test          | null          |
+| NETWORK_ID        | Network ID of connected CodeChain           | cc, bc, tc                  | null          |
```
Comment (ScarletBlue) [inline indexer-monitor/README.md]: of connected CodeChain node? Maybe should be more specific here?

File: indexer-monitor/README.md [hunk 14]
```diff
@@ -6,3 +6,12 @@ Checks whether the indexer is working properly.
...
+| INDEXER_URL       | CodeChain Indexer API URL                   | http://localhost:9001       | null          |
+| SENDGRID_API_KEY  | An API Key for alarm                        | xxx                         | null          |
```
Comment (ScarletBlue) [inline indexer-monitor/README.md]: alarms

File: indexer-monitor/README.md [hunk 15]
```diff
@@ -6,3 +6,12 @@ Checks whether the indexer is working properly.
...
+| SENDGRID_TO       | An email address that alarm will be sent to | codechain@example.com       | null          |
```
Comment (ScarletBlue) [inline indexer-monitor/README.md]: An email address that alarms will be sent to

File: indexer-monitor/README.md [hunk 16]
```diff
@@ -6,3 +6,12 @@ Checks whether the indexer is working properly.
...
+| SLACK_WEBHOOK_URL | A webhook URL for alarm                     | http://abcd.example.com/x/y | null          |
```
Comment (ScarletBlue) [inline indexer-monitor/README.md]: alarms

File: juggle-ccc/README.md [hunk 17]
```diff
@@ -1,3 +1,17 @@
 ### Juggle CCC
 Juggle CCC is not a monitoring tool, but reports whether the CCC Juggling is working properly and shows the status of the accounts currently being juggled.
 * (info level) Report the amount of CCC retained and how much you used as a fee once per day.
+
+### Environment variables
+
+| Key               | Description                                        | Example values                              | Default value |
+| ----------------- | -------------------------------------------------- | ------------------------------------------- | ------------- |
+| NETWORK_ID        | Network ID of connected CodeChain                  | cc, bc, tc                                  | null          |
```
Comment (ScarletBlue) [inline juggle-ccc/README.md]: same as above.

File: juggle-ccc/README.md [hunk 18]
```diff
@@ -1,3 +1,17 @@
...
+| RPC_URL           | CodeChain HTTP JSON RPC URL                        | http://localhost:8080                       | null          |
+| LEFT_ADDRESS      | Platform address of the left account               | bccqy7aeyc0qryehxgtx9y4vthhg7u95rad2ymlya7j | null          |
```
Comment (ScarletBlue) [inline juggle-ccc/README.md]: The platform address

File: juggle-ccc/README.md [hunk 19]
```diff
@@ -1,3 +1,17 @@
...
+| LEFT_PASSPHRASE   | A passphrase that is used to encrypt left account  | xxxx                                        | null          |
```
Comment (ScarletBlue) [inline juggle-ccc/README.md]: A passphrase used to encrypt the left account

File: juggle-ccc/README.md [hunk 20]
```diff
@@ -1,3 +1,17 @@
...
+| RIGHT_ADDRESS     | Platform address of the right account              | bccqyp5ragyz8v4mha22asnrxvy6dt2hc7yecylqy6d | null          |
```
Comment (ScarletBlue) [inline juggle-ccc/README.md]: The platform address of the right account

File: juggle-ccc/README.md [hunk 21]
```diff
@@ -1,3 +1,17 @@
...
+| RIGHT_PASSPHRASE  | A passphrase that is used to encrypt right account | xxxx                                        | null          |
```
Comment (ScarletBlue) [inline juggle-ccc/README.md]: The passphrase used to encrypt the right account

File: juggle-ccc/README.md [hunk 22]
```diff
@@ -1,3 +1,17 @@
...
+| SENDGRID_API_KEY  | An API Key for alarm                               | xxx                                         | null          |
```
Comment (ScarletBlue) [inline juggle-ccc/README.md]: An API key for alarms

File: juggle-ccc/README.md [hunk 23]
```diff
@@ -1,3 +1,17 @@
...
+| SENDGRID_TO       | An email address that alarm will be sent to        | codechain@example.com                       | null          |
```
Comment (ScarletBlue) [inline juggle-ccc/README.md]: alarms

File: monitor/README.md [hunk 24]
```diff
@@ -1,10 +1,22 @@
...
+7. (info level) Sends a report if all nodes are working fine and return to a state of agreement.
+8. (info level) A notification is sent everyday, reporting that CodeChain is in working order.
+
+### Environment variables
+
+| Key                      | Description                                 | Example values              | Default value                                                               |
+| ------------------------ | ------------------------------------------- | --------------------------- | --------------------------------------------------------------------------- |
+| NODE_ENV                 | Use "production" to reduce mistakes         | production OR test          | null                                                                        |
+| RPC_URL                  | CodeChain HTTP JSON RPC URL                 | http://localhost:8080       | https://corgi-rpc.codechain.io in corgi https://rpc.codechain.io in mainnet |
+| VIEW_ALERT_LEVEL         | See above the alarm 3                       | 3                           | null                                                                        |
```
Comment (ScarletBlue) [inline monitor/README.md]: See above alarm 3

File: monitor/README.md [hunk 25]
```diff
@@ -1,10 +1,22 @@
...
+| SLEEP_STREAK_ALERT_LEVEL | See above the alarm number 6                | 5                           | null                                                                        |
```
Comment (ScarletBlue) [inline monitor/README.md]: See above alarm 6

File: monitor/README.md [hunk 26]
```diff
@@ -1,10 +1,22 @@
...
+| SENDGRID_API_KEY         | An API Key for alarm                        | xxx                         | null                                                                        |
```
Comment (ScarletBlue) [inline monitor/README.md]: alarms

File: monitor/README.md (hunk 25 above)
Comment (ScarletBlue) [reply monitor/README.md]: or alarm number 6?

File: monitor/README.md (hunk 24 above)
Comment (ScarletBlue) [reply monitor/README.md]: or alarm number 3?

File: monitor/README.md [hunk 27]
```diff
@@ -1,10 +1,22 @@
...
+| SENDGRID_TO              | An email address that alarm will be sent to | codechain@example.com       | null                                                                        |
```
Comment (ScarletBlue) [inline monitor/README.md]: alarms

File: monitor/README.md [hunk 28]
```diff
@@ -1,10 +1,22 @@
...
+| SLACK_WEBHOOK_URL        | A webhook URL for alarm                     | http://abcd.example.com/x/y | null                                                                        |
```
Comment (ScarletBlue) [inline monitor/README.md]: alarms

File: auto-self-nominate/README.md (hunk 3 above)
Comment (majecty) [reply auto-self-nominate/README.md]: Aha, "end" is a verb!

File: indexer-monitor/README.md (hunk 13 above)
Comment (majecty) [reply indexer-monitor/README.md]: Good. "connected CodeChan" is obscure.

**Comment by majecty** [comment]
@ScarletBlue I fixed them.

**Pull request merged**